src/app/components/pages/IsaacCompetition/IsaacCompetition.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from "react";
+import { SITE_SUBJECT_TITLE } from "../../../services";
+import { TitleAndBreadcrumb } from "../../elements/TitleAndBreadcrumb";
+import { Button, Col, Container, Row } from "reactstrap";
+import { Link } from "react-router-dom";
+import content from "./content";
+
+export const IsaacCompetition = () => {
+  useEffect(() => {
+    document.title = "Isaac " + SITE_SUBJECT_TITLE;
+  }, []);
+
+  const CompetitionButton = () => {
+    const competitionInterestButton = [{ to: "/competition-sign-up", label: "Express your interest" }];
+
+    return (
+      <Row>
+        {competitionInterestButton.map(({ to, label }) => (
+          <Col xs={12} className="py-1" key={to}>
+            <Button size="lg" tag={Link} to={to} block className="primary-button text-light">
+              {label}
+            </Button>
+          </Col>
+        ))}
+      </Row>
+    );
+  };
+
+  return (
+    <>
+      <Container>
+        <TitleAndBreadcrumb currentPageTitle="Isaac Competition" breadcrumbTitleOverride="Isaac Competition" />
+      </Container>
+      <div id="competition-headline-section">
+        <Container className="pt-4 z1">
+          <Row>
+            <h1 className="primary-heading pl-3">National Computer Science Competition</h1>
+            <Col xs={12} md={6} lg={8} className="pb-3">
+              <p className="mt-4 body-text">{content.section1.header.section}</p>
```

Comment:
You can break this down even more to look something like this:
`{section1.header.section}`
by adding this at the top of the file: 
`const { section1 } = content;`